tests/auth_helpers.py

```diff
@@ -86,6 +86,18 @@ def run_auth_flow(auth_url, brand="CHEVROLET"):
     driver.find_element_by_id("password").send_keys("password")
     sign_in_button.click()
 
+    # For Volkswagen Pin Flow
+    if brand == "VOLKSWAGEN":
+        # Click the first vehicle compatible with EV endpoints. *Should* be an eGolf
+        driver.find_element_by_css_selector(".input-button-custom.checkbox").click()
+
+        # Enter Pin 1-1-1-1
+        driver.find_element_by_id("f2-pin-input-number-1").send_keys("1")
+        driver.find_element_by_id("f2-pin-input-number-2").send_keys("1")
+        driver.find_element_by_id("f2-pin-input-number-3").send_keys("1")
+        driver.find_element_by_id("f2-pin-input-number-4").send_keys("1")
```

Comment:
nit: can you convert it into a loop and just have the `driver.find_element_by_id("f2-pin-input-number-<loop-value>").send_keys("1")` once ?